paper-api/src/main/java/org/bukkit/ExplosionResult.java

```diff
@@ -10,14 +10,14 @@ public enum ExplosionResult {
 
     /**
      * Represents an explosion where no change took place.
-     *
+     * <br>
      * This is the case when {@link org.bukkit.GameRule#MOB_GRIEFING} is
-     * disabled.
+     * disabled or the {@link org.bukkit.enchantments.Enchantment#WIND_BURST}.
```

Comment:
This is wrong, on my end the enchantment use TRIGGER_BLOCK? like the wind charge

Reply:
Unless im wrong the enchatment call the explode effect not the trigger thing

Reply:
Yes but each explode effect can define the type there, here is the wind burst definition:
![image](https://github.com/user-attachments/assets/6e972131-989d-40c3-8b83-9fe282d44729)